### ISSUE TITLE
Incorrect comment detection

### ISSUE THREAD

**Issue by kinghuang** [Eiara/HCL.novaextension#7]
Text in strings are incorrectly parsed for comment markers. For example, the following text treats the `/*` and `*/` in the path component strings as the beginning and end of comments.

**Code:**

```HCL
locals {
  datasource_files = {
    for file in fileset("${path.module}/provisioning", "/datasources/*.yml"):
    file => "${path.module}/provisioning/${file}"
  }
  
  dashboard_provider_files = {
    for file in fileset("${path.module}/provisioning", "/dashboards/*.yml"):
    file => "${path.module}/provisioning/${file}"
  }

  dashboard_definition_files = {
    for file in fileset("${path.module}/provisioning", "/dashboards/**/*.json"):
    file => "${path.module}/provisioning/${file}"
  }
}
```

**Nova Screenshot:**

![image](https://user-images.githubusercontent.com/27340/100667787-323ca900-3318-11eb-8b8f-9aaaa1530267.png)

**Comment by aurynn** [comment]
Hey, sorry for the delay on this.

It should be fixed as of 0.1.5, could you confirm?
<img width="991" alt="Screen Shot 2021-02-04 at 2 08 48 PM" src="https://user-images.githubusercontent.com/165099/106830272-96059880-66f2-11eb-8746-26d7df87f1b0.png">

**Comment by aurynn** [comment]
Assuming it's fixed - please reopen if it's still an issue.
